README.md

```diff
@@ -141,13 +141,41 @@ Execution failed for task ':app:mergeDebugNativeLibs'.
 
 It means you have a transitive dependency where two libraries depend on OpenSSL and are generating a `libcrypto.so` file. You can get around this issue by adding the following in your `app/build.gradle`:
 
+<h4>
```

Comment:
Can you replace this with markdown?  `###` or similar

Reply:
Well, with markdown we can't inline those fancy images/icons, so I think in that case it's good :)

Nice lil detail @gabimoncha